src/tokens/_shape.ts

```diff
@@ -0,0 +1,11 @@
+import type {TokenGroup} from './tokens';
+
+export const shape: TokenGroup = {
+  'border-radius-05': '2px',
+  'border-radius-1': '4px',
+  'border-radius-2': '8px',
+  'border-radius-3': '12px',
+  'border-radius-4': '16px',
+  'border-radius-5': '20px',
+  'border-radius-6': '50%',
```

Comment:
I think this should still be:
```suggestion
  'border-radius-full': '50%',
```
or even 
```suggestion
  'border-radius-half': '50%',
```
I'll leave the naming up to you but I feel like it's part of a different group than the numbered values

Reply:
Yeah I was wondering about this—glad you brought it up! I like `'border-radius-half':` in case we have a 100% value at some point.